frontend/src/app/modules/boards/board/board-partitioned-page/board-list-container.component.ts

```diff
@@ -22,6 +22,7 @@ import {BoardListCrossSelectionService} from "core-app/modules/boards/board/boar
 import {filter} from "rxjs/operators";
 import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-actions/board-actions-registry.service";
 import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
+import { WorkPackageStatesInitializationService } from 'core-app/components/wp-list/wp-states-initialization.service';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/14ff584ec2db54c1b1bb925f0be1e3ec383ca507/frontend/src/app/modules/boards/board/board-partitioned-page/board-list-container.component.ts#L26"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>